code/game/objects/items/storage/medkit.dm

```diff
@@ -320,6 +320,56 @@
 	)
 	generate_items_inside(items_inside,src)
 
+/obj/item/storage/medkit/coroner
+	name = "compact coroner's medkit"
+	desc = "A smaller medical kit designed primarily for assisting in dissecting the deceased, rather than treating the living."
+	icon = 'icons/obj/storage/medkit.dmi'
+	icon_state = "compact_coronerkit"
+	inhand_icon_state = "coronerkit"
+	var/max_slots = 6
+	var/max_total_storage = 6
+
+/obj/item/storage/medkit/coroner/Initialize(mapload)
+	. = ..()
+	atom_storage.max_specific_storage = WEIGHT_CLASS_SMALL //lowering this so it can't hold the autopsy scanner
+	atom_storage.max_slots = max_slots
+	atom_storage.max_total_storage = max_total_storage
+
+/obj/item/storage/medkit/coroner/PopulateContents()
+	if(empty)
+		return
+	var/static/items_inside = list(
+		/obj/item/reagent_containers/cup/bottle/formaldehyde = 1,
+		/obj/item/reagent_containers/medigel/sterilizine = 1,
+		/obj/item/reagent_containers/blood = 1,
+		/obj/item/bodybag = 2,
+		/obj/item/reagent_containers/syringe = 1,
+	)
+	generate_items_inside(items_inside,src)
+
+/obj/item/storage/medkit/coroner/large
+	name = "coroner's medkit"
+	desc = "A medical kit designed primarily for assisting in dissecting the deceased, rather than treating the living."
+	icon = 'icons/obj/storage/medkit.dmi'
+	icon_state = "coronerkit"
+	inhand_icon_state = "coronerkit"
+	max_slots = 12
+	max_total_storage = 24
+
+/obj/item/storage/medkit/coroner/large/PopulateContents()
+	if(empty)
+		return
+	var/static/items_inside = list(
+		/obj/item/reagent_containers/cup/bottle/formaldehyde = 1,
+		/obj/item/reagent_containers/medigel/sterilizine = 1,
+		/obj/item/toy/crayon/white = 1,
+		/obj/item/reagent_containers/blood = 1,
+		/obj/item/bodybag = 2,
+		/obj/item/reagent_containers/syringe = 1,
+		/obj/item/folder/white = 1,//for storing autopsy reports from the scanner
```

Comment:
I really hope Coroners use the filing cabine we put in their morgue instead, there detectives are able to access it if they need to, and I thought it would be more flavorful. Up to you though

Reply:
oh i didn't realize they had folders in their cabinets, i'll cut that. cool if they keep the clipboard in their pockets?

Reply:
they don't have folders, you put the papers in the cabinets themselves to store the ppaers.
The clipboard is fine, I think the folder is fine, it's really just storing paper really, nothing that much of a deal.

Reply:
well, alright then. i can add the folder back in! glad you're okay with the changes too, put a lot of (feverish) energy into this PR over the past couple days lmfao